### DISCUSSION TITLE
What the terminal is in README?

### DISCUSSION THREAD

**Discussion by jqhr** [gsuuon/llm.nvim#38] [Q&A] [answered]
I like this terminal, looks pretty.
What is it?

**Comment by gsuuon** [comment, marked as answer]
This is [Wezterm](https://github.com/wez/wezterm) :). I posted parts of my config [here](https://github.com/wez/wezterm/discussions/628#discussioncomment-5735102).

**Reply by jqhr** [reply]
Good.
Thank you
